ui/selectmenu.js

```diff
@@ -150,7 +154,6 @@ return $.widget( "ui.selectmenu", {
 					// If the item was selected via a click, the text selection
 					// will be destroyed in IE
 					that._setSelection();
-
```

Comment:
Removing this line makes the comment harder to follow. You no longer know that it only applies to one line.